template-reactless/my_component/__init__.py

```diff
@@ -34,7 +34,7 @@
     # replace the `url` param with `path`, and point it to to the component's
     # build directory:
     parent_dir = os.path.dirname(os.path.abspath(__file__))
-    build_dir = os.path.join(parent_dir, "frontend/build")
+    build_dir = os.path.join(parent_dir, "frontend-react/build")
     _component_func = components.declare_component("my_component", path=build_dir)
```

Comment:
This should be either `my_component` or we should replace `my_component` everywhere with proper name

Reply:
What do you mean by proper name?

Reply:
Should not this replaced with {{ cookiecutter.import_name }}?